core/src/test/java/org/mapfish/print/attribute/ScalebarAttributeTest.java

```diff
@@ -90,7 +90,7 @@ public void testAttributesFromJson() throws Exception {
         assertEquals("m", value.unit);
         assertTrue(value.geodetic);
         assertTrue(value.lockUnits);
-        assertEquals("Arial", value.font);
+        assertEquals("Liberation Sans", value.font);
```

Comment:
I prefer to provide a download link like: https://www.fontsquirrel.com/fonts/Liberation-Sans
But I don't see where to add it...